paasta_tools/paasta_metastatus.py

```diff
@@ -180,14 +180,19 @@ def assert_quorum_size(state):
 
 
 def assert_extra_slave_data(mesos_state):
-    rows = [('Hostname', 'CPU free', 'RAM free')]
-    for slave in get_extra_mesos_slave_data(mesos_state):
-        rows.append((
-            slave['hostname'],
-            '%.2f' % slave['free_resources']['cpus'],
-            '%.2f' % slave['free_resources']['mem'],
-        ))
-    return ('\n'.join(('    %s' % row for row in format_table(rows)))[2:], True)
+    extra_slave_data = get_extra_mesos_slave_data(mesos_state)
+    if extra_slave_data:
+        rows = [('Hostname', 'CPU free', 'RAM free')]
+        for slave in extra_slave_data:
+            rows.append((
+                slave['hostname'],
+                '%.2f' % slave['free_resources']['cpus'],
+                '%.2f' % slave['free_resources']['mem'],
+            ))
+        output = '\n'.join(('    %s' % row for row in format_table(rows)))[2:]
+    else:
+        output = '  No mesos slaves on this cluster'
```

Comment:
Also say it with an exclamation mark!!!

Be more specific like: "No mesos slaves registered in this cluster!!!"